Java 10

When building the runtime on a fresh Arch install, I got the following errors:

```
# nathan @ samus in ~/Projects/melt/silver/runtime/java on git:develop o [23:31:41] 
$ ant
Buildfile: /home/nathan/Projects/melt/silver/runtime/java/build.xml

init:

build:
    [javac] Compiling 168 source files to /home/nathan/Projects/melt/silver/generated/bin
    [javac] /home/nathan/Projects/melt/silver/runtime/java/src/common/rawlib/RawXML.java:368: error: PairListNSContext is not abstract and does not override abstract method getPrefixes(String) in NamespaceContext
    [javac] 	public static class PairListNSContext implements NamespaceContext {
    [javac] 	              ^
    [javac] /home/nathan/Projects/melt/silver/runtime/java/src/common/rawlib/RawXML.java:414: error: getPrefixes(String) in PairListNSContext cannot implement getPrefixes(String) in NamespaceContext
    [javac] 		public Iterator<?> getPrefixes(String namespaceURI) {
    [javac] 		                   ^
    [javac]   return type Iterator<?> is not compatible with Iterator<String>
    [javac] /home/nathan/Projects/melt/silver/runtime/java/src/common/rawlib/RawXML.java:413: error: method does not override or implement a method from a supertype
    [javac] 		@Override
    [javac] 		^
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 3 errors

BUILD FAILED
/home/nathan/Projects/melt/silver/runtime/java/build.xml:28: Compile failed; see the compiler error output for details.

Total time: 4 seconds
```

I'm using javac 10.0.1, which is probably the issue; Java 9 works on the same machine.

--- a/runtime/java/src/common/rawlib/RawXML.java
+++ b/runtime/java/src/common/rawlib/RawXML.java
@@ -411,7 +411,7 @@ public final class RawXML {
 		}
 
 		@Override
-		public Iterator<?> getPrefixes(String namespaceURI) {
+		public Iterator getPrefixes(String namespaceURI) {
 			throw new UnsupportedOperationException();
 		}
 

